### PR TITLE
i18n(es) feat: Mention bun package manager support

### PR DESCRIPTION
**Pull request by K1ri86**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change?
Added and updated the Spanish translation for github.mdx

### CODE REVIEW

File: src/content/docs/es/guides/deploy/github.mdx [hunk 1]
```diff
@@ -90,7 +90,7 @@ Astro mantiene la acción oficial `withastro/action` para desplegar tu proyecto
     :::
 
     :::caution
-    La [action oficial de Astro](https://github.com/withastro/action) busca un archivo lockfile para detectar tu gestor de paquetes (`npm`, `yarn` o `pnpm`). Debes tener el archivo `package-lock.json`, `yarn.lock` o `pnpm-lock.yaml`, generado automáticamente por tu gestor de paquetes, en tu repositorio.
+    La [action oficial de Astro](https://github.com/withastro/action) busca un archivo lockfile para detectar tu gestor de paquetes (`npm`, `yarn`, `pnpm`, o `bun`). Debes tener el archivo `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml`, o `bun.lockb`, generado automáticamente por tu gestor de paquetes, en tu repositorio.
```
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/github.mdx:93]:
```suggestion
    La [action oficial de Astro](https://github.com/withastro/action) busca un archivo lockfile para detectar tu gestor de paquetes (`npm`, `yarn`, `pnpm`, o `bun`). Debes tener el archivo `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml` o `bun.lockb`, generado automáticamente por tu gestor de paquetes, en tu repositorio.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

File: src/content/docs/es/guides/deploy/github.mdx (hunk 1 above)
Comment (K1ri86) [reply src/content/docs/es/guides/deploy/github.mdx:93]: Gracias Waxer ahora mismo lo hago  <3

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 2]
```diff
@@ -203,6 +211,33 @@ Por defecto, `@astrojs/cloudflare` generará un archivo `_routes.json` con regla
...
+
+Para usar estas APIs, la página o el punto final deben ser renderizados en el lado del servidor (no pre-renderizados) y deben utilizar la sintaxis de importación `import {} from 'node:*'`.
+
+```js
+// pages/api/endpoint.js
+export const prerender = false;
+import { Buffer } from 'node:buffer';
+```
+
+Además, deberás habilitar la Bandera de Compatibilidad en Cloudflare. La configuración de esta bandera puede variar según donde implementes tu sitio Astro.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:237]:
```suggestion
Además, deberás habilitar la Bandera de Compatibilidad en Cloudflare. La configuración de esta bandera puede variar según donde implementes tu sitio de Astro.
```

File: src/content/docs/es/guides/deploy/github.mdx (hunk 1 above)
Comment (Waxer59) [inline src/content/docs/es/guides/deploy/github.mdx:93]:
```suggestion
    La [action oficial de Astro](https://github.com/withastro/action) busca un archivo lockfile para detectar tu gestor de paquetes (`npm`, `yarn`, `pnpm` o `bun`). Debes tener el archivo `package-lock.json`, `yarn.lock`, `pnpm-lock.yaml` o `bun.lockb`, generado automáticamente por tu gestor de paquetes, en tu repositorio.
```

File: src/content/docs/es/guides/integrations-guide/cloudflare.mdx [hunk 3]
```diff
@@ -203,6 +211,33 @@ Por defecto, `@astrojs/cloudflare` generará un archivo `_routes.json` con regla
...
+*   Buffer
+*   Diagnostics Channel
+*   EventEmitter
+*   path
+*   process
+*   Streams
+*   StringDecoder
+*   util
+
+Para usar estas APIs, la página o el punto final deben ser renderizados en el lado del servidor (no pre-renderizados) y deben utilizar la sintaxis de importación `import {} from 'node:*'`.
```
Comment (Waxer59) [inline src/content/docs/es/guides/integrations-guide/cloudflare.mdx:229]:
```suggestion
Para usar estas APIs, la página o el endpoint debe ser renderizado en el lado del servidor (no pre-renderizado) y debe utilizar la sintaxis de importación `import {} from 'node:*'`.
```

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Waxer59 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**